index.html

```diff
@@ -177,21 +200,21 @@ <h4>Listening to the Hash</h4>
             </p>
 
             <small class="col-12">
-                <pre>window.onhashchange = () => hashExample.setAttribute("deer-id", window.location.hash.substr(1))</pre>
+                <pre>window.onhashchange = () => hashExample.setAttribute(DEER.ID, window.location.hash.substr(1))</pre>
             </small>
 
 
             <div class="card col-6">
                 <header>List Template with Links</header>
-                <div class="deer-view" deer-link="#" deer-id="http://devstore.rerum.io/v1/id/5c9d15afe4b0a44e13e6170c" deer-list="itemListElement" deer-template="list"></div>
+                <deer-view deer-link="#" deer-id="http://devstore.rerum.io/v1/id/5c9d15afe4b0a44e13e6170c" deer-list="itemListElement" deer-template="list"></deer-view>
                 <footer class="text-primary"><img src="https://icongr.am/fontawesome/bolt.svg?size=18"> Click on a link in the list to update the rendered element on the right.</footer>
             </div>
 
             <div class="card col-6">
                 <header>Load from Hash</header>
-                <div class="col deer-view" id="hashExample" deer-template="entity">
+                <deer-view class="col" id="hashExample" deer-template="entity">
```

Comment:
This did not update when the has changed

Reply:
*it works on my machine* Confirm perhaps that the `onhashchange` listener is active at the bottom of the page? Maybe some other pageload error stopped before it ran?